Package.swift

```diff
@@ -17,6 +17,10 @@ let package = Package(
         //    name: "PayPalNativeCheckout",
         //    targets: ["PayPalNativeCheckout"]
         //),
+        .library(
+            name: "PayPalUI",
+            targets: ["PayPalUI"]
```

Comment:
Just a general question, but will we include other UI Components in this module in the future? For example Venmo buttons, text fields, validation logic potentially, or the Drop-in? If so, calling it `PayPalUI` will almost certainly be confusing if we want this to be the entry point for merchants who don't want to handle visual elements themselves.

Reply:
Yeah I'm not sure I'd say the name is subject to change. We called it PayPalUI on Android so I just went for it.

Reply:
Ah, gotcha gotcha. Yeah some of those may end up pretty small. Like a Venmo module would probably just be 1 button? Unless there are other UI elements for Venmo that I don't know about. 🤷‍♂️ Wonder if it's also worth exploring NXO decoupling the smart payment buttons into it's own module vs us duplicating what already exists in the package size of NXO but isn't being used.

But yes, totally agree on differentiating the name of the SDK!